pages/design/atomic-template-empty.md

```diff
@@ -1,35 +1,50 @@
 ---
+id: 
 name:
 rule_type: atomic
 
 description: |
   This rule checks ...
 
-success_criterion:
-  - x.x.x # (Name of Success Criterion)
-
-test_aspects:
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:x.x.x: # <Name of Success Criterion> (A | AA | AAA)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied | further testing needed
+    inapplicable: satisfied | further testing needed
+    
+  aria11:some-id: # <Some heading in WAI-ARIA>
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied | further testing needed
+    inapplicable: satisfied | further testing needed
+
+  wcag-technique:H123: # <Technique title>
+    forConformance: false
+    failed: not satisfied
+    passed: satisfied | further testing needed
+    inapplicable: satisfied | further testing needed
+
```

Comment:
@WilcoFiers, can we add the possibility for if nothing is required as well? 

Or, if it all becomes too much to have in the template, just write a basic template for how to do this, and then have each of the more detailed examples (WCAG/WAI-ARIA/techniques/nothing) in a separate place, maybe on a page on the website that we link out to in a comment in this section.

Reply:
I think the possibility of having no `accessibility_requirements` is something that should be explained elsewhere, perhaps in the documentation and wiki that is put together by @nitedog.

Reply:
Once that page is in place, it can be cross-referred here. This work should not withhold this PR.